jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/KubernetesHelper.java

```diff
@@ -509,7 +513,7 @@ public static void handleKubernetesClientException(KubernetesClientException e,
     }
 
     public static Set<HasMetadata> loadResources(File manifest) throws IOException {
-        Object dto = ResourceUtil.load(manifest, KubernetesResource.class);
+        Object dto = ResourceUtil.loadKubernetesResourceList(manifest);
```

Comment:
This will break the if-else cause in line 521 dto will never be an instance of `Template`

Reply:
I'm tracking this down to be only used on the undeploy phase goal

Reply:
https://github.com/eclipse/jkube/pull/534/files#r568642847

Method names are really confusing here.